### DISCUSSION TITLE
Allow Users to click 'Reboot' Status to reboot nodes

### DISCUSSION THREAD

**Discussion by jaypfe** [Security-Onion-Solutions/securityonion#12765] [Ideas]
Hey guys!

I absolutely love the new feature to be able to restart nodes from the grid page. It saves so so much time. 

I had a thought today when my whole grid was showing that it was needing restarted, what if the status symbol when it showed 'reboot' in blue was actually clickable and actioned the reboot. That way you would not have to open each node, you could just rapid fire down the list. 

Thanks again for this amazing product, and the continued work! 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/107776997/f86ae33e-5aee-4392-8394-90c6b90fbd3a)

**Comment by cm-ops** [comment]
For many nodes, if you want to reboot at the same time, it would probably be better to use salt to reboot them all.
